dffml/source/csv.py

```diff
@@ -8,11 +8,20 @@
 
 from ..repo import Repo
 from .memory import MemorySource
-from .file import FileSource
+from .file import FileSource, FileSourceConfig
+from typing import NamedTuple, Dict
```

Comment:
Let's move this `typing` import out of the `dffml` imports and up to where we import from the rest of the standard library